src/styles.scss

```diff
@@ -2,6 +2,7 @@
 
 /* Importing Bootstrap SCSS file. */
 @import '~bootstrap/scss/bootstrap';
+@import '~bootstrap-icons/font/bootstrap-icons.css';
```

Comment:
We'll have to look at if this matters if it's css, since we use scss for the project. If there is an scss one we should use it, but if not we'll make sure everything works fine with this as a css import.